src/components/Input/Input.tsx

```diff
@@ -1,16 +1,34 @@
-import React from "react";
+import React, { useState } from "react";
 
 import ConfigProps from "src/interfaces/config-props";
 
-import { confirmIcon, emailIcon, passwordIcon, userIcon } from "./inputIcons";
+import {
+  confirmIcon,
+  emailIcon,
+  passwordIcon,
+  userIcon,
+  visibleIcon,
+} from "./inputIcons";
 
 import "./style.scss";
 
 interface InputProps extends ConfigProps {}
 
-const Input: React.FC<InputProps> = ({ name, placeholder, type, icon }) => {
+const Input: React.FC<InputProps> = ({
+  name,
+  placeholder,
+  type,
+  leftIcon,
+  rightIcon,
+}) => {
+  const [visible, setVisible] = useState(false);
+
+  const toggleVisible = () => {
+    
+  };
```

Comment:
Do you plan to implement this in the future

Reply:
?

Reply:
Yes, i think i will use this function.